README.md

```diff
@@ -14,8 +14,8 @@ modules: {
     }
   },
   'apostrophe-forms-widgets': {},
-  'apostrophe-forms-google-address-field-widgets'
+  'apostrophe-forms-google-address-field-widgets': {}
 }
 ```
 
-The user will need a valid Google API key for the suggestions displayed by Google to work.
+The user will need a valid Google API key for the suggestions displayed by Google to work as stated in Google Maps Platform documentation for Places API: [https://developers.google.com/places/web-service/autocomplete](https://developers.google.com/places/web-service/autocomplete) A premium plan is needed to obtain a key from Google.
```

Comment:
Is billing required? Or is it only required if you hit a certain limit? I think you are simply limited to lower usage without billing.

Reply:
It is yes. They offer you $200 when subscribing though.

Reply:
It seems like this requires the Maps Javascript API as well. We should be very clear which need to be enabled.

I don't think Google phrases it like "premium plan." I suggest phrasing it as saying something like "you must connect a billing account to the Google API console project."

Reply:
Thanks for the phrasing